src/webapp/components/survey-list/SurveyList.tsx

```diff
@@ -74,6 +77,15 @@ export const SurveyList: React.FC<SurveyListProps> = ({ surveyFormType }) => {
                         surveys
                     ) && (
                         <ButtonWrapper>
+                            {surveyFormType === "PPSPatientRegister" && (
+                                <TextField
+                                    label={i18n.t("Search Patient")}
+                                    helperText={i18n.t("Filter by patient id or code")}
+                                    value={patientFilterKeyword}
+                                    onChange={e => setPatientFilterKeyword(e.target.value)}
```

Comment:
if setPatientFilterKeyword is done when the form is of PPSPatientRegister type, then the user goes to another form type, the patientFilterKeyword  will still be set. Correct? Can you test this? I think we need to reset patientFilterKeyword to empty whenever the formtype is not PPSPatientRegister.

Reply:
I've tested and every time a user enters a new survey with some keyword entered, it is reset on render, so I think it should be fine.